core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -292,6 +292,7 @@ public String toString() {
           .setDescription("The directory containing files used to configure Alluxio.")
           .setIgnoredSiteProperty(true)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.ALL)
```

Comment:
default scope is `Scope.ALL` right?

Reply:
Yeah, but this can distinguish the property keys that do not consider about its scope / undefined and the property keys that are actually of scope.ALL. And remind following developers to always set scope.

Reply:
Can we just enforce that all property keys define a scope? I think it would be better that way

Reply:
I prefer not to because there are maybe some property keys generated from templates.